akka-projection-jdbc/src/test/scala/akka/projection/jdbc/JdbcProjectionSpec.scala

```diff
@@ -300,19 +313,15 @@ class JdbcProjectionSpec
       }
 
       withClue("check - all offsets were seen") {
-        val offset = offsetStore.readOffset[Long](projectionId).futureValue.value
-        offset shouldBe 6L
+        assertOffsetIs(6L)
       }
     }
 
     "skip failing events when using RecoveryStrategy.skip" in {
       val entityId = UUID.randomUUID().toString
-      val projectionId = genRandomProjectionId()
+      implicit val projectionId = genRandomProjectionId()
```

Comment:
I took a shortcut and made `projectionId` an implicit argument in the new methods (which I kind of like, TBH)